src/Debugger/Impl/rtvs/R/grid.R

```diff
@@ -72,3 +72,79 @@ grid.dput <- function(obj) {
     json;
 }
 
+getEnvRepr <- function(env, startEnvLevel) {
```

Comment:
What is the purpose of this function in general? If it's just to get the name of the environment, why not just eval `environmentName` directly? And what is `startEnvLevel`, and `level` in the return value?